x-pack/plugins/data_visualizer/public/application/index_data_visualizer/hooks/use_overall_stats.ts

```diff
@@ -116,72 +142,63 @@ export function useOverallStats<TParams extends OverallStatsSearchStrategyParams
       abortSignal: abortCtrl.current.signal,
       sessionId: searchStrategyParams?.sessionId,
     };
-    const nonAggregatableOverallStats$ =
-      nonAggregatableFields.length > 0
-        ? forkJoin(
-            nonAggregatableFields.map((fieldName: string) =>
-              data.search
-                .search<IKibanaSearchRequest, IKibanaSearchResponse>(
-                  {
-                    params: checkNonAggregatableFieldExistsRequest(
-                      index,
-                      searchQuery,
-                      fieldName,
-                      timeFieldName,
-                      earliest,
-                      latest,
-                      runtimeFieldMap
-                    ),
-                  },
-                  searchOptions
-                )
-                .pipe(
-                  switchMap((resp) => {
-                    return of({
-                      ...resp,
-                      rawResponse: { ...resp.rawResponse, fieldName },
-                    } as IKibanaSearchResponse);
-                  })
-                )
-            )
-          )
-        : of(undefined);
+
+    const nonAggregatableFieldsObs = nonAggregatableFields.map((fieldName: string) =>
+      data.search
+        .search<IKibanaSearchRequest, IKibanaSearchResponse>(
+          {
+            params: checkNonAggregatableFieldExistsRequest(
+              index,
+              searchQuery,
+              fieldName,
+              timeFieldName,
+              earliest,
+              latest,
+              runtimeFieldMap
+            ),
+          },
+          searchOptions
+        )
+        .pipe(
+          switchMap((resp) => {
+            return of({
+              ...resp,
+              rawResponse: { ...resp.rawResponse, fieldName },
+            } as IKibanaSearchResponse);
+          })
```

Comment:
you should use a `map` operator instead 
```suggestion
          map((resp) => {
            return {
              ...resp,
              rawResponse: { ...resp.rawResponse, fieldName },
            } as IKibanaSearchResponse);
          }
```

Reply:
Updated here https://github.com/elastic/kibana/pull/124898/commits/6e5b6014eb02d7a53008c58d1d103d0b4fbe10ad